include/xrpl/protocol/LedgerFormats.h

```diff
@@ -189,6 +189,9 @@ enum LedgerSpecificFlags {
 
     // ltCREDENTIAL
     lsfAccepted = 0x00010000,
+
+    // ltVAULT
+    lsfVaultPrivate = 0x00010000,
 };
```

Comment:
I guess we do not have flag `lsfShareNonTransferable` because, for this purpose, we will rely on `lsfMPTCanTransfer` instead ?

Reply:
also need flag `lsfFrozen`

Reply:
Answer to `lsfShareNonTransferable`: correct. Please refer to the [spec](https://github.com/XRPLF/XRPL-Standards/blob/f253b1b0bd755b217153090ecb01037105cb57b5/XLS-0065d-single-asset-vault/README.md#21611-mptokenissuance-values).

Answer to `lsfFrozen`: there is no vault freeze, thus no `lsfFrozen`.